src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidationServiceConstants.java

```diff
@@ -8,7 +8,9 @@
  *
  */
 public class FigaroValidationServiceConstants {
-	
+
+	private FigaroValidationServiceConstants() {}
```

Comment:
empty?

Reply:
yes it's a private constructor to prevent anyone for initializing the constants object

Reply:
thank you java-sensei. my skills are not elite. 🥋